pkg/apis/core/types.go

```diff
@@ -2634,16 +2634,21 @@ type PodSecurityContext struct {
 	// If unspecified, the container runtime will allocate a random SELinux context for each
 	// container.  May also be set in SecurityContext.  If set in
 	// both SecurityContext and PodSecurityContext, the value specified in SecurityContext
-	// takes precedence for that container.
+	// takes precedence.
 	// +optional
 	SELinuxOptions *SELinuxOptions
 	// The UID to run the entrypoint of the container process.
 	// Defaults to user specified in image metadata if unspecified.
 	// May also be set in SecurityContext.  If set in both SecurityContext and
-	// PodSecurityContext, the value specified in SecurityContext takes precedence
-	// for that container.
+	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsUser *int64
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
```

Comment:
If the Docker image doesnt have group specified, then it default to 0, else it will use the gid specified in the docker image, i will reword to include that detail